reana_db/models.py

```diff
@@ -145,14 +146,19 @@ def _get_health_status(usage, limit):
 
         quota_usage = 0
         quota_limit = 0
+        quota_human_readable_usage = ""
         for user_resource in self.resources:
             if user_resource.resource.type_ == resource_type:
                 quota_usage += user_resource.quota_used
                 quota_limit += user_resource.quota_limit
+                quota_human_readable_usage = ResourceUnit.human_readable_unit(
+                    user_resource.resource.unit, user_resource.quota_used
+                )
 
         return {
             "usage": quota_usage,
             "limit": quota_limit,
+            "human_readable_usage": quota_human_readable_usage,
```

Comment:
I believe we also need a `human_readable_limit` to display properly in the clients in a "X KB out of Y MB" way.

Reply:
Makes sense, what do you think of changing the format of the dictionary to something like follows?
```python
        return {
            "usage": {
                "raw": quota_usage,
                "human_readable": quota_human_readable_usage,
            },
            "limit": {
                "raw": quota_limit,
                "human_readable": quota_human_readable_limit,
            },
            "health": _get_health_status(quota_usage, quota_limit),
        }
```

Reply:
IMHO nested data structures may be harder to consume in the client side. It could also lead to some problems if there is something wrong with the data and client tries to access nested object (additional checks needed)

Reply:
With the new [optional chaining ES6 operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) we don't need to do those additional checks anymore, if the key doesn't exist we will get `undefined`.